app/tests/spec/lib/url.js

```diff
@@ -63,6 +63,16 @@ function (chai, _, Url) {
         assert.equal(Url.objToSearchString({}), '');
       });
     });
+
+    describe('removeParamFromSearchString', function () {
+      it('removes a param', function () {
+        assert.equal(Url.removeParamFromSearchString('foo', '?foo=one&bar=two'), '?bar=two');
+      });
+
+      it('returns the same string if param is not included', function () {
+        assert.equal(Url.removeParamFromSearchString('foo', '?bar=one&baz=two'), '?bar=one&baz=two');
+      });
+    });
```

Comment:
Can you add a check to ensure an empty query string does not cause an explosion?